lib/models/sender_service.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * A SenderService collection stores all the services that have
+ * contacted a specific user (fiscalCode) using the notification APIs
+ */
+import { pick } from "italia-ts-commons/lib/types";
+
+import * as DocumentDb from "documentdb";
+import * as t from "io-ts";
+
+import { tag } from "italia-ts-commons/lib/types";
+
+import * as DocumentDbUtils from "../utils/documentdb";
+import { DocumentDbModel } from "../utils/documentdb_model";
+
+import { FiscalCode } from "../api/definitions/FiscalCode";
+
+import { DateFromString } from "italia-ts-commons/lib/dates";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { ServiceId } from "../api/definitions/ServiceId";
+
+export const SENDER_SERVICE_COLLECTION_NAME = "sender-services";
+export const SENDER_SERVICE_MODEL_ID_FIELD = "fiscalCode";
+export const SENDER_SERVICE_MODEL_PK_FIELD = "fiscalCode";
+
+export const SenderService = t.interface({
+  createdAt: DateFromString,
+  fiscalCode: FiscalCode,
```

Comment:
`fiscalCode` is not needed here (?)

Reply:
ok sorry this is the fiscal code of the recipient... we should probably give to these attributes a more specific name, like `recipient: FiscalCode` and `sender: ServiceId`

Reply:
if the path would be /services?recipient=fiscalCode I think that we should maintain serviceId (as a Service sub-property)

Reply:
ok for me for serviceId in that context is not ambiguous